scripts/new_project.kts

```diff
@@ -238,9 +262,22 @@ object NewProject {
         }
     }
 
-    private fun executeCommand(command: String) {
+    /**
+     * Execute a shell command
+     *
+     * Runtime.getRuntime().exec(String) will partition a command automatically, based on white spaces.
+     * If a file path contains a white space, it will not be able to find the file and result in an error.
```

Comment:
How about generally this a bit?
```suggestion
     * If a file path contains any white spaces, it will not be able to find the file and result in an error.
```

Reply:
@lydiasama Updated 😁